responders/src/API/EMBC.Responders.API/Controllers/RegistrationsController.cs

```diff
@@ -247,31 +247,56 @@ public async Task<ActionResult<RegistrationResult>> CreateFileNote(string fileId
         }
 
         /// <summary>
-        /// Updates a File Note
+        /// Updates a File Note's content
         /// </summary>
-        /// <param name="fileId">fileId</param>
         /// <param name="noteId">noteId</param>
         /// <param name="note">note</param>
-        /// <returns>newly created note id</returns>
-        [HttpPost("files/{fileId}/notes/{noteId}")]
+        /// <returns>updated note id</returns>
+        [HttpPost("notes/{noteId}/content")]
```

Comment:
not needed to be that specific, also, you may not find the note and should return 404. Also, the files/{id} was a good thing, any reason to remove it?